packages/discord.js/src/structures/UserSelectMenuInteraction.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+const { Collection } = require('@discordjs/collection');
+const SelectMenuInteraction = require('./SelectMenuInteraction');
+/**
+ * Represents a {@link ComponentType.UserSelect} select menu interaction.
+ * @extends {SelectMenuInteraction}
+ */
+class UserSelectMenuInteraction extends SelectMenuInteraction {
+    constructor(client, data) {
+        super(client, data);
+        /**
+         * Collection of the selected users
+         * @type {Collection<Snowflake, User|APIUser>}
+         */
+        this.users = new Collection();
+        for (const user of Object.values(data.data.resolved.users)) {
+            this.users.set(user.id, this.client.users._add(user) ?? user);
+          }
+
+          if (data.data.resolved.members) {
+                /**
+                 * Collection of the selected users
+                 * @type {Collection<Snowflake, GuildMember|APIGuildMember>?}
+                 */
+                this.members = new Collection();
+                for (const member of Object.values(data.data.resolved.members)) {
+                    this.members.set(member.id, this.guild?.members._add(member) ?? member);
+                }
+            } else {
+                this.members = null;
+            }
```

Comment:
Are you sure we can select not only guild members? I can select only guild members.